c++/src/kj/exception.c++

```diff
@@ -38,6 +38,13 @@
 #include <execinfo.h>
 #endif
 
+#if _WIN32
+#define WIN32_LEAN_AND_MEAN
+#include <windows.h>
+#include "windows-sanity.h"
```

Comment:
I recommend:

```
#define WIN32_LEAN_AND_MEAN
#define NOGDI     // Inhibit ERROR define
#define NOMINMAX  // Inhibit min/max defines
#include <windows.h>
```

Then this file will compile in MSVC. The GDI in NOGDI stands for Graphics Device Interface -- presumably cruft we can safely excise in this .c++ file.